gradio/utils.py

```diff
@@ -38,7 +38,7 @@
 
 def version_check():
     try:
-        current_pkg_version = pkg_resources.require("gradio")[0].version
```

Comment:
We need to back these changes out due to importlib not being available on 3.7 right?

Reply:
oh, that's correct 😄